BaseFiles/Common/html/ext/zwave/_nodesetup.js

```diff
@@ -543,21 +581,21 @@ HG.WebApp.GroupModules.SensorMultiLevelParameterGet = function () {
 function zwave_AssociationGet(nodeid, groupid, callback) {
     $.get('/' + HG.WebApp.Data.ServiceKey + '/HomeAutomation.ZWave/' + nodeid + '/Association.Get/' + groupid + '/', function (data) {
         if (typeof callback != 'undefined' && callback != null) {
-            callback(data.ResponseValue);
```

Comment:
Please keep back the use of ResponseText object. See also my other note:
https://github.com/genielabs/mig-service-dotnet/pull/7#discussion_r56712929